### DISCUSSION TITLE
Migation in custom schema postgres

### DISCUSSION THREAD

**Discussion by driver005** [medusajs/medusa#5806] [Q&A]
Trying to migrate with medusa-cli migrate to custom schema called medusa.
My config:
/** @type {import('@medusajs/medusa').ConfigModule["projectConfig"]} */
const projectConfig = {
	jwtSecret: process.env.JWT_SECRET,
	cookieSecret: process.env.COOKIE_SECRET,
	store_cors: STORE_CORS,
	database_url: DATABASE_URL,
	admin_cors: ADMIN_CORS,
	database_schema: 'medusa', // define schema
	// Uncomment the following lines to enable REDIS
	// redis_url: REDIS_URL
};

But instead all except for the migration tabel are int the public schema 
witch makes it also imposable to revert the migration

**Comment by srindom** [comment]
So your database has: `medusa.migrations` but `public.product` and not `medusa.product`?

**Reply by driver005** [reply]
Yes exactly
